universal-login-sdk/src/core/services/ExecutionFactory.ts

```diff
@@ -1,11 +1,11 @@
-import {MessageStatus, SignedMessage, stringifySignedMessageFields, ensureNotFalsy} from '@unilogin/commons';
+import {MessageStatus, SignedMessage, stringifySignedMessageFields, ensureNotFalsy, MineableStatus} from '@unilogin/commons';
 import {RelayerApi} from '../../integration/http/RelayerApi';
 import {MissingMessageHash} from '../utils/errors';
 import {MineableFactory} from './MineableFactory';
 
 export interface Execution {
-  waitForTransactionHash: () => Promise<MessageStatus>;
-  waitToBeSuccess: () => Promise<MessageStatus>;
+  waitForTransactionHash: () => Promise<MineableStatus>;
+  waitToBeSuccess: () => Promise<MineableStatus>;
```

Comment:
It should be MessageStatus (Execution => MessageStatus, DeployingWallet => DeploymentStatus). Probably we can add sanitizer for MessageStatus and cast it in ExecutionFactory